src/app/shared/components/dual-slider/dual-slider.component.ts

```diff
@@ -15,24 +15,53 @@ import {
 import { Options } from 'ng5-slider';
 
 
+/**
+ * Component containing a button that when clicked will show a slider popover.
+ */
 @Component({
   selector: 'ccf-dual-slider',
   templateUrl: './dual-slider.component.html',
   styleUrls: ['./dual-slider.component.scss']
 })
 export class DualSliderComponent implements OnDestroy, OnChanges {
+  /**
+   * Reference to the template for the slider popover.
+   */
   @ViewChild(CdkPortal, { static: true }) popoverPortal: CdkPortal;
+
+  /**
+   * Reference to the popover element.
+   * This is undefined until the slider popover is initialized.
+   */
   @ViewChild('popover', { read: ElementRef, static: false }) popoverElement: ElementRef;
+
   @Input() label: string;
```

Comment:
@inputs and @Outputs are the highest priority thing to document when dealing with components. Please document.